docs/application/native/guides/ui/efl/wearable/component-gengrid.md

```diff
@@ -0,0 +1,222 @@
+| `drag,start,right`        | The item in the grid is dragged (not scrolled) right. | `Elm_Object_Item`                        |
+| `drag,stop`               | The item in the grid has stopped being dragged. | `Elm_Object_Item`                        |
+| `drag`                    | The item in the grid is being dragged.   | `Elm_Object_Item` object that contains the dragged item |
+| `scroll,drag,start`       | Dragging the content is started.         | `NULL`                                   |
+| `scroll,drag,stop`        | Dragging the content is stopped.         | `NULL`                                   |
+| `scroll`                  | Scrolling is ongoing.                    | `NULL`                                   |
+| `edge,top`                | The gengrid is scrolled to the top edge. | `NULL`                                   |
+| `edge,bottom`             | The gengrid is scrolled to the bottom edge. | `NULL`                                   |
+| `edge,left`               | The gengrid is scrolled to the left edge. | `NULL`                                   |
+| `edge,right`              | The gengrid is scrolled to the right edge. | `NULL`                                   |
+| `moved`                   | The item is moved in the reorder mode.   | `Elm_Object_Item` object that contains the moved item |
+| `pressed`                 | The item is pressed by mouse down.       | `Elm_Object_Item` object that contains the pressed item |
+| `released`                | The item is released by mouse up.        | `Elm_Object_Item` object that contains the released item |
+| `clicked,right`           | The item is right-clicked.               | Right-clicked gengrid item               |
+| `longpressed`             | The item is pressed for a certain amount of time. By default, it is 1 second. |               -                           |
+| `changed`                 | The item is added, removed, resized, or moved, and the gengrid is resized or has `horizontal` property changes. |       -                                   |
+| `scroll,anim,start`       | The scrolling animation starts.          |         -                                 |
+| `scroll,anim,stop`        | The scrolling animation stops.           |         -                                 |
+| `scroll,page,changed`     | The visible page changes.                |           -                               |
+| `item,reorder,anim,start` | The reorder animation starts.            |        -                                  |
+| `item,reorder,anim,stop`  | The reorder animation stops.             |   -                                       |
+
+> [!Note]
+> The signal list in the API reference can be more extensive, but only the above signals are actually supported in Tizen.
+
+> [!Note]
+> Except as noted, this content is licensed under [LGPLv2.1+](http://opensource.org/licenses/LGPL-2.1).
```

Comment:
The note style is wrong. Please update it like below.

```suggestion
> [!NOTE]
> The signal list in the API reference can be more extensive, but only the above signals are actually supported in Tizen.

> [!NOTE]
> Except as noted, this content is licensed under [LGPLv2.1+](http://opensource.org/licenses/LGPL-2.1).
```

In https://docs1.stg.tizen.org/staging/1332/application/native/guides/ui/efl/wearable/component-gengrid/,
you can see the note tag is not working.

![image](https://user-images.githubusercontent.com/9958374/98499589-4fd4a080-228d-11eb-9ca8-ba12e103ea94.png)

Example of working note style : 
![image](https://user-images.githubusercontent.com/9958374/98499637-6ed33280-228d-11eb-90dc-de9f83ed3af6.png)